controllers/grants/index.js

```diff
@@ -265,6 +265,15 @@ router.get('/:id', injectCopy('funding.pastGrants.search'), async (req, res, nex
     try {
         const data = await grantsService.getById({ id: req.params.id, locale: req.i18n.getLocale() });
 
+        let caseStudy;
```

Comment:
Not sure how I feel about this `setHeroLocals` function. Could this whole thing be inside an `injectCaseStudy` function?